kale/tests/unit_tests/static_analysis/test_dependencies.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+
+from kale.static_analysis.dependencies import pyflakes_report
+
+
+@pytest.mark.parametrize("code,target", [
+    ('', []),
+    ('a = b', ['b']),
+    ('a = foo(b)', ['foo', 'b']),
+    ('a = b \n foo(b)', []),
```

Comment:
I may haven't understood this, but shouldn't this return `['b', 'foo']` ?

The indentation seems incorrect, and this might mess it up. If you write
```python
'a = b\nfoo(b)'
```
which is equivalent to
```python
"""
a = b
foo(b)
"""
```
instead of
```python
"""
a = b
 foo(b)
"""
```
then it actually does return `['b', 'foo']`